src/pages/AddressInfoPage/index.tsx

```diff
@@ -57,127 +59,106 @@ type ParamTypes = {
   id: string
 }
 
-const TransactionInfoPage = () => {
+type FetchedEntity = 'balance' | 'txNumber' | 'assets' | 'transactions'
```

Comment:
nit: I have a few renames to suggest, cause I struggled parsing the code. How do you feel about renaming this to simply `AddressPropertyName`? (see below for other renames to make more sense).